src/constants.ts

```diff
@@ -12,6 +12,7 @@ const RELATIVE_EXPORT_PATH = core.getInput('relative_export_path');
 const RELATIVE_PROJECT_PATH = core.getInput('relative_project_path');
 const SHOULD_CREATE_RELEASE = core.getInput('create_release') === 'true';
 const UPDATE_WINDOWS_ICONS = core.getInput('update_windows_icons') === 'true';
+const USE_PRESET_EXPORT_PATH = core.getInput('use_preset_export_path') === 'false';
```

Comment:
This does not set a default value, rather this just converts the inputs to boolean values for use later. This line should be `=== 'true'` like the others, and the input and its default value need to be defined in [this file](https://github.com/firebelley/godot-export/blob/master/action.yml). Let me know if anything is unclear!